components/VcItemContent.tsx

```diff
@@ -160,16 +168,52 @@ export const VcItemContent: React.FC<VcItemContentProps> = (props) => {
             !props.verifiableCredential ? Theme.Styles.loadingContainer : null
           }>
           <Column>
-            {uin ? getDetails(t('uin'), uin, props.verifiableCredential) : null}
-            {vid ? getDetails(t('vid'), vid, props.verifiableCredential) : null}
+            {uin ? (
+              <Column margin="0 0 20 0">
+                <Text
+                  testID="uin"
```

Comment:
This is uin label, actual uin renderer should also have testID

Reply:
Refactored this.